jjb/dynamic/scale-ci_install_azure.yml

```diff
@@ -0,0 +1,293 @@
+    - password:
+        default: ""
+        description: pull secret
+        name: OPENSHIFT_INSTALL_PULL_SECRET
+    - password:
+        default: ""
+        description: Registry token to talk to quay registry
+        name: OPENSHIFT_INSTALL_QUAY_REGISTRY_TOKEN
+    - string:
+        default: "registry.svc.ci.openshift.org"
+        description: Image registry to use to pull down the images
+        name: OPENSHIFT_INSTALL_IMAGE_REGISTRY
+    - password:
+        default: ""
+        description: Registry token to talk to the image registry registry being used
+        name: OPENSHIFT_INSTALL_REGISTRY_TOKEN
+    - bool:
+        default: false
+        description: Build the installer binary from source code
+        name: OPENSHIFT_INSTALL_INSTALLER_FROM_SOURCE
+    - string:
+        default: "master"
+        description: Source version to use to build the installer binary
+        name: OPENSHIFT_INSTALL_INSTALLER_FROM_SOURCE_VERSION
+    - string:
+        default: "/root/.go"
+        description: gopath
+        name: GOPATH
+    - password:
+        default: ''
+        description: The id of the subscription to GCP to use. Use `az account show` to find id
+        name: AZURE_SUBSCRIPTION_ID
+    - password:
+        default: ''
+        description: The GCP tenantId from `az account show`
+        name: AZURE_TENANT_ID
+    - password:
+        default: ''
+        description: The GCP service principal id to use for this cluster. Create a service principal (one time) with `az ad sp create-for-rbac --role Owner --name test-sp`
+        name: AZURE_SERVICE_PRINCIPAL_CLIENT_ID
+    - password:
+        default: ""
+        description: The service principal password. This should be shown in the return on the `az ad sp create-for-rbac` command ran above to create the service principal
+        name: AZURE_SERVICE_PRINCIPAL_CLIENT_SECRET
+    - password:
+        default: ""
+        description: The resource group in which the base domain is located
+        name: AZURE_BASE_DOMAIN_RESOURCE_GROUP_NAME
+    - string:
+        default: ""
+        description: "The GCP region to install on to. Example: `centralus`"
+        name: AZURE_REGION
+    - string:
+        default: ""
+        description: Base domain for OCP install.
+        name: OPENSHIFT_BASE_DOMAIN
+    - string:
+        default: ""
+        description: openshift cluster name.
+        name: OPENSHIFT_CLUSTER_NAME
+    - string:
+        default: "3"
+        description: openshift master node count.
+        name: OPENSHIFT_MASTER_COUNT
+    - string:
+        default: "5"
+        description: openshift worker node count.
+        name: OPENSHIFT_WORKER_COUNT
+    - string:
+        default: "Standard_D4s_v3"
+        description: openshift master vm size.
+        name: OPENSHIFT_MASTER_VM_SIZE
+    - string:
+        default: "Standard_D4s_v3"
+        description: openshift worker vm size.
+        name: OPENSHIFT_WORKER_VM_TYPE
```

Comment:
s/OPENSHIFT_WORKER_VM_TYPE/OPENSHIFT_WORKER_VM_SIZE/

Reply:
Fixed it, thanks.